### PR TITLE
ci: Update CI workflow to only build what has changed

### PR DESCRIPTION
**Pull request by jakemhiller**
### Overview

Example run: https://app.circleci.com/pipelines/github/Codecademy/client-modules/8343/workflows/33995123-372c-4439-90d7-d6c9c27ce260

1. Runs jest tests against source code instead of expecting pre-built packages. Requires some setup if we add more packages that require other monorepo packages, but it's pretty simple to set up.
2. Splits out CI tasks based on whether they can run with only the built packages that have changed, or whether they need all packages to be built. Right now only the deploy tasks require all packages to be built.

What it doesn't do:

1. All tests are still run, I went down a rabbit hole trying to get only the relevant changed tests to run and it either didn't run enough tests to be valid (jest `onlyChanged` options), or was too complex to set up (jest projects) (maybe requires a jest upgrade).

**Previous Workflow:**
Screen Shot 2020-12-03 at 6 10 05 PM

**New Workflow**
Screen Shot 2020-12-04 at 10 44 45 AM

### PR Checklist

- [ ] Related to designs:
- [ ] Related to JIRA ticket: ABC-123
- [ ] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fc80b24a4fdfa00922b4aa7--gamut-preview.netlify.app>https://5fc80b24a4fdfa00922b4aa7--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fc80b24a4fdfa00922b4aa7>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fc80cabeb1a83009c768c96--gamut-preview.netlify.app>https://5fc80cabeb1a83009c768c96--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fc80cabeb1a83009c768c96>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fc8106c57cfa00158716fc3--gamut-preview.netlify.app>https://5fc8106c57cfa00158716fc3--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fc8106c57cfa00158716fc3>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fc8133f56db45014ee33efd--gamut-preview.netlify.app>https://5fc8133f56db45014ee33efd--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fc8133f56db45014ee33efd>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fc90cec478dae0baeac9a5a--gamut-preview.netlify.app>https://5fc90cec478dae0baeac9a5a--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fc90cec478dae0baeac9a5a>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fc91091d97f1a0ef726b46e--gamut-preview.netlify.app>https://5fc91091d97f1a0ef726b46e--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fc91091d97f1a0ef726b46e>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@21.0.5-alpha.9a4b92.0<br/>@codecademy/webpack-config@5.2.1-alpha.9a4b92.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@21.0.5-alpha.c6267f.0<br/>@codecademy/webpack-config@5.2.1-alpha.c6267f.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fc91488e379ab152b5703ab--gamut-preview.netlify.app>https://5fc91488e379ab152b5703ab--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fc91488e379ab152b5703ab>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fc957fb87cd8b0e7c21c728--gamut-preview.netlify.app>https://5fc957fb87cd8b0e7c21c728--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fc957fb87cd8b0e7c21c728>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@21.0.5-alpha.a53f7d.0<br/>@codecademy/webpack-config@5.2.1-alpha.a53f7d.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@21.0.5-alpha.a59e1e.0<br/>@codecademy/webpack-config@5.2.1-alpha.a59e1e.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fc95865ef7da00d3133d503--gamut-preview.netlify.app>https://5fc95865ef7da00d3133d503--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fc95865ef7da00d3133d503>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@21.0.5-alpha.247c03.0<br/>@codecademy/webpack-config@5.2.1-alpha.247c03.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fc959067e589310314aa2ad--gamut-preview.netlify.app>https://5fc959067e589310314aa2ad--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fc959067e589310314aa2ad>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.11.2-alpha.ad792c.0<br/>@codecademy/gamut-labs@7.0.5-alpha.ad792c.0<br/>@codecademy/gamut-styles@7.4.9-alpha.ad792c.0<br/>@codecademy/gamut-system@0.1.6-alpha.ad792c.0<br/>@codecademy/gamut@19.0.4-alpha.ad792c.0<br/>@codecademy/markdown-overrides@0.3.17-alpha.ad792c.0<br/>@codecademy/styleguide@21.0.5-alpha.ad792c.0<br/>@codecademy/tracking@0.1.3-alpha.ad792c.0<br/>@codecademy/webpack-config@5.2.1-alpha.ad792c.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.11.2-alpha.7e31d3.0<br/>@codecademy/gamut-labs@7.0.5-alpha.7e31d3.0<br/>@codecademy/gamut-styles@7.4.9-alpha.7e31d3.0<br/>@codecademy/gamut-system@0.1.6-alpha.7e31d3.0<br/>@codecademy/gamut@19.0.4-alpha.7e31d3.0<br/>@codecademy/markdown-overrides@0.3.17-alpha.7e31d3.0<br/>@codecademy/styleguide@21.0.5-alpha.7e31d3.0<br/>@codecademy/tracking@0.1.3-alpha.7e31d3.0<br/>@codecademy/webpack-config@5.2.1-alpha.7e31d3.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.11.2-alpha.e18aa2.0<br/>@codecademy/gamut-labs@7.0.5-alpha.e18aa2.0<br/>@codecademy/gamut-styles@7.4.9-alpha.e18aa2.0<br/>@codecademy/gamut-system@0.1.6-alpha.e18aa2.0<br/>@codecademy/gamut@19.0.4-alpha.e18aa2.0<br/>@codecademy/markdown-overrides@0.3.17-alpha.e18aa2.0<br/>@codecademy/styleguide@21.0.5-alpha.e18aa2.0<br/>@codecademy/tracking@0.1.3-alpha.e18aa2.0<br/>@codecademy/webpack-config@5.2.1-alpha.e18aa2.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fc95e257e58931c024aa05d--gamut-preview.netlify.app>https://5fc95e257e58931c024aa05d--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fc95e257e58931c024aa05d>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.11.2-alpha.63789a.0<br/>@codecademy/gamut-labs@7.0.5-alpha.63789a.0<br/>@codecademy/gamut-styles@7.4.9-alpha.63789a.0<br/>@codecademy/gamut-system@0.1.6-alpha.63789a.0<br/>@codecademy/gamut@19.0.4-alpha.63789a.0<br/>@codecademy/markdown-overrides@0.3.17-alpha.63789a.0<br/>@codecademy/styleguide@21.0.5-alpha.63789a.0<br/>@codecademy/tracking@0.1.3-alpha.63789a.0<br/>@codecademy/webpack-config@5.2.1-alpha.63789a.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fc95fdcadf6bc1a5d96dfbf--gamut-preview.netlify.app>https://5fc95fdcadf6bc1a5d96dfbf--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fc95fdcadf6bc1a5d96dfbf>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.11.2-alpha.59dd42.0<br/>@codecademy/gamut-labs@7.0.5-alpha.59dd42.0<br/>@codecademy/gamut-styles@7.4.9-alpha.59dd42.0<br/>@codecademy/gamut-system@0.1.6-alpha.59dd42.0<br/>@codecademy/gamut@19.0.4-alpha.59dd42.0<br/>@codecademy/markdown-overrides@0.3.17-alpha.59dd42.0<br/>@codecademy/styleguide@21.0.5-alpha.59dd42.0<br/>@codecademy/tracking@0.1.3-alpha.59dd42.0<br/>@codecademy/webpack-config@5.2.1-alpha.59dd42.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fc961a27b043518d79bec71--gamut-preview.netlify.app>https://5fc961a27b043518d79bec71--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fc961a27b043518d79bec71>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.11.2-alpha.4e40ba.0<br/>@codecademy/gamut-labs@7.0.5-alpha.4e40ba.0<br/>@codecademy/gamut-styles@7.4.9-alpha.4e40ba.0<br/>@codecademy/gamut-system@0.1.6-alpha.4e40ba.0<br/>@codecademy/gamut@19.0.4-alpha.4e40ba.0<br/>@codecademy/markdown-overrides@0.3.17-alpha.4e40ba.0<br/>@codecademy/styleguide@21.0.5-alpha.4e40ba.0<br/>@codecademy/tracking@0.1.3-alpha.4e40ba.0<br/>@codecademy/webpack-config@5.2.1-alpha.4e40ba.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fc96b086ca15627e224ee02--gamut-preview.netlify.app>https://5fc96b086ca15627e224ee02--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fc96b086ca15627e224ee02>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.11.2-alpha.39b85e.0<br/>@codecademy/gamut-labs@7.0.5-alpha.39b85e.0<br/>@codecademy/gamut-styles@7.4.9-alpha.39b85e.0<br/>@codecademy/gamut-system@0.1.6-alpha.39b85e.0<br/>@codecademy/gamut@19.0.4-alpha.39b85e.0<br/>@codecademy/markdown-overrides@0.3.17-alpha.39b85e.0<br/>@codecademy/styleguide@21.0.5-alpha.39b85e.0<br/>@codecademy/tracking@0.1.3-alpha.39b85e.0<br/>@codecademy/webpack-config@5.2.1-alpha.39b85e.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fc96c23425c6b25030ad07e--gamut-preview.netlify.app>https://5fc96c23425c6b25030ad07e--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fc96c23425c6b25030ad07e>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.11.2-alpha.500f21.0<br/>@codecademy/gamut-labs@7.0.5-alpha.500f21.0<br/>@codecademy/gamut-styles@7.4.9-alpha.500f21.0<br/>@codecademy/gamut-system@0.1.6-alpha.500f21.0<br/>@codecademy/gamut@19.0.4-alpha.500f21.0<br/>@codecademy/markdown-overrides@0.3.17-alpha.500f21.0<br/>@codecademy/styleguide@21.0.5-alpha.500f21.0<br/>@codecademy/tracking@0.1.3-alpha.500f21.0<br/>@codecademy/webpack-config@5.2.1-alpha.500f21.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fc96d4887cd8b262621c5a1--gamut-preview.netlify.app>https://5fc96d4887cd8b262621c5a1--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fc96d4887cd8b262621c5a1>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fc96e3001780e266a1953e3--gamut-preview.netlify.app>https://5fc96e3001780e266a1953e3--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fc96e3001780e266a1953e3>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.11.2-alpha.73e281.0<br/>@codecademy/gamut-labs@7.0.5-alpha.73e281.0<br/>@codecademy/gamut-styles@7.4.9-alpha.73e281.0<br/>@codecademy/gamut-system@0.1.6-alpha.73e281.0<br/>@codecademy/gamut@19.0.4-alpha.73e281.0<br/>@codecademy/markdown-overrides@0.3.17-alpha.73e281.0<br/>@codecademy/styleguide@21.0.5-alpha.73e281.0<br/>@codecademy/tracking@0.1.3-alpha.73e281.0<br/>@codecademy/webpack-config@5.2.1-alpha.73e281.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.11.2-alpha.aaf6db.0<br/>@codecademy/gamut-labs@7.0.5-alpha.aaf6db.0<br/>@codecademy/gamut-styles@7.4.9-alpha.aaf6db.0<br/>@codecademy/gamut-system@0.1.6-alpha.aaf6db.0<br/>@codecademy/gamut@19.0.4-alpha.aaf6db.0<br/>@codecademy/markdown-overrides@0.3.17-alpha.aaf6db.0<br/>@codecademy/styleguide@21.0.5-alpha.aaf6db.0<br/>@codecademy/tracking@0.1.3-alpha.aaf6db.0<br/>@codecademy/webpack-config@5.2.1-alpha.aaf6db.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fc97183aaa7a7009093cdbf--gamut-preview.netlify.app>https://5fc97183aaa7a7009093cdbf--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fc97183aaa7a7009093cdbf>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fc97347aaa7a700a493cd0e--gamut-preview.netlify.app>https://5fc97347aaa7a700a493cd0e--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fc97347aaa7a700a493cd0e>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fca54ba747b1d013cc7d6f0--gamut-preview.netlify.app>https://5fca54ba747b1d013cc7d6f0--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fca54ba747b1d013cc7d6f0>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.11.2-alpha.080445.0<br/>@codecademy/gamut-labs@7.0.5-alpha.080445.0<br/>@codecademy/gamut-styles@7.4.9-alpha.080445.0<br/>@codecademy/gamut-system@0.1.6-alpha.080445.0<br/>@codecademy/gamut@19.0.4-alpha.080445.0<br/>@codecademy/markdown-overrides@0.3.17-alpha.080445.0<br/>@codecademy/styleguide@21.0.5-alpha.080445.0<br/>@codecademy/tracking@0.1.3-alpha.080445.0<br/>@codecademy/webpack-config@5.2.1-alpha.080445.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.11.2-alpha.a86484.0<br/>@codecademy/gamut-labs@7.0.5-alpha.a86484.0<br/>@codecademy/gamut-styles@7.4.9-alpha.a86484.0<br/>@codecademy/gamut-system@0.1.6-alpha.a86484.0<br/>@codecademy/gamut@19.0.4-alpha.a86484.0<br/>@codecademy/markdown-overrides@0.3.17-alpha.a86484.0<br/>@codecademy/styleguide@21.0.5-alpha.a86484.0<br/>@codecademy/tracking@0.1.3-alpha.a86484.0<br/>@codecademy/webpack-config@5.2.1-alpha.a86484.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fca55b5d4c15800cb745719--gamut-preview.netlify.app>https://5fca55b5d4c15800cb745719--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fca55b5d4c15800cb745719>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.11.2-alpha.f0277e.0<br/>@codecademy/gamut-labs@7.0.5-alpha.f0277e.0<br/>@codecademy/gamut-styles@7.4.9-alpha.f0277e.0<br/>@codecademy/gamut-system@0.1.6-alpha.f0277e.0<br/>@codecademy/gamut@19.0.4-alpha.f0277e.0<br/>@codecademy/markdown-overrides@0.3.17-alpha.f0277e.0<br/>@codecademy/styleguide@21.0.5-alpha.f0277e.0<br/>@codecademy/tracking@0.1.3-alpha.f0277e.0<br/>@codecademy/webpack-config@5.2.1-alpha.f0277e.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fca58ff170311008d065920--gamut-preview.netlify.app>https://5fca58ff170311008d065920--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fca58ff170311008d065920>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fca5d79bf10e40096698191--gamut-preview.netlify.app>https://5fca5d79bf10e40096698191--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fca5d79bf10e40096698191>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.11.2-alpha.59e3ad.0<br/>@codecademy/gamut-labs@7.0.5-alpha.59e3ad.0<br/>@codecademy/gamut-styles@7.4.9-alpha.59e3ad.0<br/>@codecademy/gamut-system@0.1.6-alpha.59e3ad.0<br/>@codecademy/gamut@19.0.4-alpha.59e3ad.0<br/>@codecademy/markdown-overrides@0.3.17-alpha.59e3ad.0<br/>@codecademy/styleguide@21.0.5-alpha.59e3ad.0<br/>@codecademy/tracking@0.1.3-alpha.59e3ad.0<br/>@codecademy/webpack-config@5.2.1-alpha.59e3ad.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.11.2-alpha.d56736.0<br/>@codecademy/gamut-labs@7.0.5-alpha.d56736.0<br/>@codecademy/gamut-styles@7.4.9-alpha.d56736.0<br/>@codecademy/gamut-system@0.1.6-alpha.d56736.0<br/>@codecademy/gamut@19.0.4-alpha.d56736.0<br/>@codecademy/markdown-overrides@0.3.17-alpha.d56736.0<br/>@codecademy/styleguide@21.0.5-alpha.d56736.0<br/>@codecademy/tracking@0.1.3-alpha.d56736.0<br/>@codecademy/webpack-config@5.2.1-alpha.d56736.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fca60f04bd789008fb18ce2--gamut-preview.netlify.app>https://5fca60f04bd789008fb18ce2--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fca60f04bd789008fb18ce2>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fca6b8ceccfcc0d06c70799--gamut-preview.netlify.app>https://5fca6b8ceccfcc0d06c70799--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fca6b8ceccfcc0d06c70799>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.11.2-alpha.68e47a.0<br/>@codecademy/gamut-labs@7.0.5-alpha.68e47a.0<br/>@codecademy/gamut-styles@7.4.9-alpha.68e47a.0<br/>@codecademy/gamut-system@0.1.6-alpha.68e47a.0<br/>@codecademy/gamut@19.0.4-alpha.68e47a.0<br/>@codecademy/markdown-overrides@0.3.17-alpha.68e47a.0<br/>@codecademy/styleguide@21.0.5-alpha.68e47a.0<br/>@codecademy/tracking@0.1.3-alpha.68e47a.0<br/>@codecademy/webpack-config@5.2.1-alpha.68e47a.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.11.2-alpha.8073c6.0<br/>@codecademy/gamut-labs@7.0.6-alpha.8073c6.0<br/>@codecademy/gamut-styles@7.5.1-alpha.8073c6.0<br/>@codecademy/gamut-system@0.1.6-alpha.8073c6.0<br/>@codecademy/gamut@19.0.5-alpha.8073c6.0<br/>@codecademy/markdown-overrides@0.3.18-alpha.8073c6.0<br/>@codecademy/styleguide@21.0.6-alpha.8073c6.0<br/>@codecademy/tracking@0.1.3-alpha.8073c6.0<br/>@codecademy/webpack-config@5.2.1-alpha.8073c6.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fca7080dc10531712d31203--gamut-preview.netlify.app>https://5fca7080dc10531712d31203--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fca7080dc10531712d31203>Deploy Logs</a></p>

File: packages/styleguide/.storybook/main.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ module.exports = {
     '@storybook/addon-knobs/register',
     './addons/system/preset',
   ],
-  stories: ['../stories/**/*.stories.(mdx|tsx)'],
+  stories: ['../stories/**/*.stories.@(mdx|tsx)'],
```
Comment (codecaaron) [inline packages/styleguide/.storybook/main.ts:27]: Did this resolve the blob complaint on `yarn start`?

**Review by codecaaron — COMMENTED** [review]
This looks amazing <3

File: packages/styleguide/.storybook/main.ts (hunk 1 above)
Comment (jakemhiller) [reply packages/styleguide/.storybook/main.ts:27]: yeah

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.11.2-alpha.0c0ca3.0<br/>@codecademy/gamut-labs@7.0.6-alpha.0c0ca3.0<br/>@codecademy/gamut-styles@7.5.1-alpha.0c0ca3.0<br/>@codecademy/gamut-system@0.1.6-alpha.0c0ca3.0<br/>@codecademy/gamut@19.0.5-alpha.0c0ca3.0<br/>@codecademy/markdown-overrides@0.3.18-alpha.0c0ca3.0<br/>@codecademy/styleguide@21.0.6-alpha.0c0ca3.0<br/>@codecademy/tracking@0.1.3-alpha.0c0ca3.0<br/>@codecademy/webpack-config@5.2.1-alpha.0c0ca3.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fca7520ee709317167d3acd--gamut-preview.netlify.app>https://5fca7520ee709317167d3acd--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fca7520ee709317167d3acd>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.11.2-alpha.b47f52.0<br/>@codecademy/gamut-labs@7.0.6-alpha.b47f52.0<br/>@codecademy/gamut-styles@7.5.1-alpha.b47f52.0<br/>@codecademy/gamut-system@0.1.6-alpha.b47f52.0<br/>@codecademy/gamut@19.0.5-alpha.b47f52.0<br/>@codecademy/markdown-overrides@0.3.18-alpha.b47f52.0<br/>@codecademy/styleguide@21.0.6-alpha.b47f52.0<br/>@codecademy/tracking@0.1.3-alpha.b47f52.0<br/>@codecademy/webpack-config@5.2.1-alpha.b47f52.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fca7a71dff9b000d5005314--gamut-preview.netlify.app>https://5fca7a71dff9b000d5005314--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fca7a71dff9b000d5005314>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@21.0.6-alpha.f3ba90.0<br/>@codecademy/webpack-config@5.2.1-alpha.f3ba90.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fca92391990ca00ddc926ce--gamut-preview.netlify.app>https://5fca92391990ca00ddc926ce--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fca92391990ca00ddc926ce>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@21.0.6-alpha.e4809e.0<br/>@codecademy/webpack-config@5.2.1-alpha.e4809e.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fcaa4461990ca1b49c924ec--gamut-preview.netlify.app>https://5fcaa4461990ca1b49c924ec--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fcaa4461990ca1b49c924ec>Deploy Logs</a></p>

File: packages/styleguide/.storybook/main.ts [hunk 2]
```diff
@@ -54,6 +54,15 @@ module.exports = {
       alias: {
         ...emotionless(config.resolve.alias),
         '~styleguide/blocks': path.resolve(__dirname, './components/'),
+        '@codecademy/gamut-styles$': path.resolve(
+          __dirname,
+          '../../gamut-styles/src'
+        ),
```
Comment (codecaaron) [inline packages/styleguide/.storybook/main.ts:60]: I see what you mean about this being a bit frustrating, but I don't really see any way to this dynamically.  Might want to add a bit in the readme about adding a new package maybe?

**Review by codecaaron — APPROVED** [review]
🚀 This is great.

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@21.0.6-alpha.634b7c.0<br/>@codecademy/webpack-config@5.2.1-alpha.634b7c.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fcab130b98fe32c7786851f--gamut-preview.netlify.app>https://5fcab130b98fe32c7786851f--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fcab130b98fe32c7786851f>Deploy Logs</a></p>

**Pull request merged**